resources/git-hooks/commit-msg

```diff
@@ -15,11 +15,15 @@
 # Get the directory where this script lives, i.e. resources/git-hooks
 HOOK_DIRECTORY="$(dirname "$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")")"
 
-if [ -f $(dirname "${BASH_SOURCE[0]}")/commit-msg-defs ]; then
-  . $(dirname "${BASH_SOURCE[0]}")/commit-msg-defs
-else
-  . $HOOK_DIRECTORY/commit-msg-defs
-fi
+. "$HOOK_DIRECTORY/../build/jq.inc.sh"
+. "$HOOK_DIRECTORY/commit-msg-defs"
```

Comment:
This basically reverts https://github.com/keymanapp/keyman/pull/3112. The motivation behind that change was to be able to share the implementation of this hook while giving the flexibility to have different definitions suitable for that project.

How do you suggest this hook can be shared between projects? Copy the file to the other project? But then it doesn't benefit from fixes and improvements to this hook...

Reply:
Yeah .... difficult. I don't really want to introduce a resources/scopes/scopes.json for every single repo at this point!

Reply:
Okay, I've refactored: if a `commit-msg-defs` file is present, uses that. Otherwise uses the `scopes.json` format. Currently only supports `scopes.json` from the keyman repo but I think that should be sufficient for our needs.